library/include/borealis/core/view.hpp

```diff
@@ -984,8 +984,16 @@ class View
      * The sound will only be played if the listener returned true.
      *
      * A hidden action will not show up in the bottom-right hints.
+     *
+     * Returns the identifier for the action, so it can be unregistered later on. Returns -1 if the
+     * action was not registered.
+     */
+    int registerAction(std::string hintText, enum ControllerButton button, ActionListener actionListener, bool hidden = false, enum Sound sound = SOUND_NONE);
```

Comment:
The identifier type should be hidden away from the user, with a typedef and constant for `ACTION_NONE`

`typedef int ActionIdentifier`
`#define ACTION_NONE -1`

Reply:
Done. :+1: